build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
+if %errorlevel% equ 0 (
+  set TestFixedRuntimeProjectJson=%~dp0src\Common\test-runtime\LatestDependencies\project.json
+  set defaultBuildConfig=/p:Configuration=Windows_NT_Debug
```

Comment:
For WCF package building, we always build against the dependencies we specified in the product project.json file, which is the minimum dependency. To know if there are issues with latest dependency, we run our tests against the WCF packages plus latest version dependent packages. Note, we don't rebuild WCF packages against latest here. If you worry about build break by latest dependencies (eg. API changes), our test against latest will find out. If you worry about functional break (eg. API behavior change), building against latest won't tell us that anyway and only test can.